vault_core/src/config.rs

```diff
@@ -21,22 +21,33 @@ pub struct Config {
     /// The number of vaults managed by the program
     pub num_vaults: u64,
 
+    /// The fee cap in basis points ( withdraw and deposit )
+    pub fee_cap_bps: u16,
+
+    /// The maximum amount a fee can increase per epoch in basis points
+    pub max_fee_bump_per_epoch_bps: u16,
+
     /// The bump seed for the PDA
     pub bump: u8,
 
     /// Reserved space
-    reserved: [u8; 7],
+    reserved: [u8; 11],
 }
 
 impl Config {
+    pub const DEFAULT_FEES_CAP_BPS: u16 = 3_000; // 30%
```

Comment:
let's do 2% for cap; 30% a lot to eat on deposit + withdrawal. even 2% high.

on the fee bump, i was thinking something like a rate of change as opposed to hard coded number.  for instance, 25% max rate of change per epoch starting with 10 bips:
- 10
- 12
- 15
- 19
- 24
- 30
- 38
- 47
- ...

look at MAX_WITHDRAWAL_FEE_INCREASE in the stake pool code

Reply:
Done